pygdf/series.py

```diff
@@ -306,6 +306,12 @@ def __mul__(self, other):
     def __rmul__(self, other):
         return self._rbinaryop(other, 'mul')
 
+    def __pow__(self, other):
+        if other == 2:
+            return self * self
+        else:
+            return NotImplemented
```

Comment:
This is janky.  Is there a GPU implementation somewhere that we can use?

Reply:
@mrocklin This is a WIP in https://github.com/gpuopenanalytics/libgdf/pull/94